pkg/storaged/utils.js

```diff
@@ -617,6 +617,30 @@
     return children;
 }
 
+export function get_all_children(client, path) {
+    const children = get_children(client, path);
+
+    if (client.blocks[path]) {
+        const mdraid = client.blocks[path].MDRaidMember;
+        if (mdraid != "/")
+            children.push(mdraid);
+    }
+
+    if (client.blocks_pvol[path]) {
+        const vgroup = client.blocks_pvol[path].VolumeGroup;
+        if (vgroup != "/")
+            children.push(vgroup);
+    }
+
+    if (client.blocks_stratis_blockdev[path]) {
+        const pool = client.blocks_stratis_blockdev[path].Pool;
+        if (pool != "/")
+            children.push(pool);
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230425-113621-0ab0dcb3-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)